### PR TITLE
fix(docs): IntelliJ READMEs section links

### LINKED ISSUE #799: Guestbook .md links don't work in IntelliJ
When opening the Cloud Code Java Guestbook sample in IntelliJ, the links in the table of contents do not work.

Since the links work in VS Code but not IntelliJ, it's likely that there's a formatting issue that needs to be resolved to render the README markdown correctly in IntelliJ, and the other samples' READMEs likely have this same issue in IntelliJ.

**Cloud Code in use:**
 - [Cloud Code for IntelliJ]
 - OS: macOS
 - Version 21.7.1

### PR DESCRIPTION
**Pull request by kelsk**
Replaces standard Markdown anchors (`## Section`) with html-style headers (`<h2 id="section"> Section </h2>`)

When viewing files in the Markdown Preview renderer in IntelliJ (and other JetBrains IDEs), the standard Markdown reference link format doesn't work. Links need to be styled like HTML headers. 

Fixes #799

### CODE REVIEW

**Review by iennae — APPROVED** [review]
It'd be helpful to know the why these need to get converted from markdown to html in the description. I scanned the markdown to html and it looks consistent.

File: dotnet/dotnet-guestbook/.readmes/intellij/README.md [hunk 1]
```diff
@@ -119,7 +125,7 @@ You can see how the Guestbook modules are defined by checking out the frontend's
 For more info on how to use Skaffold modules, see the [Skaffold documentation](https://skaffold.dev/docs/design/config/#multiple-configuration-support).
 
 ---
-### Run individual services with Skaffold modules
+<h3 id="run-individual-services-with-skaffold-modules"> Run individual services with Skaffold modules </h3>
```
Comment (averikitsch) [inline dotnet/dotnet-guestbook/.readmes/intellij/README.md:128]: This is the same Id as above

File: golang/go-guestbook/.readmes/intellij/README.md [hunk 2]
```diff
@@ -119,7 +125,7 @@ You can see how the Guestbook modules are defined by checking out the frontend's
 For more info on how to use Skaffold modules, see the [Skaffold documentation](https://skaffold.dev/docs/design/config/#multiple-configuration-support).
 
 ---
-### Run individual services with Skaffold modules
+<h3 id="run-individual-services-with-skaffold-modules"> Run individual services with Skaffold modules </h3>
```
Comment (averikitsch) [inline golang/go-guestbook/.readmes/intellij/README.md:128]: same here

Comment (iennae) [reply golang/go-guestbook/.readmes/intellij/README.md:128]: is it the same id or are these different files?

File: dotnet/dotnet-guestbook/.readmes/intellij/README.md (hunk 1 above)
Comment (iennae) [reply dotnet/dotnet-guestbook/.readmes/intellij/README.md:128]: oh yep, you totally caught problems with the ids! Great catch @averikitsch

Comment (kelsk) [reply dotnet/dotnet-guestbook/.readmes/intellij/README.md:128]: I'll make a separate PR to fix this - the entire "Run individual services with Skaffold modules" section is included twice, and I'll remove the duplicate from each readme

**Pull request merged**